### DISCUSSION TITLE
Is there a .js still or is documentation outdated/busted?

### DISCUSSION THREAD

**Discussion by bkardell** [KhronosGroup/KTX-Software#754] [General]
Currently several places 'point' to there being a .js/.wasm API. I was curious about this and looking for it and additional information but most of the things I find in, for example, the [wiki section "using ktx files on the web](https://github.com/KhronosGroup/KTX-Software/wiki#using-ktx-files-on-the-web) links to pages which are 404 (for example https://github.khronos.org/KTX-Software/libktx_js.html)

**Comment by MarkCallow** [comment]
Yes there is still a .js wrapper for read-only libktx.

I did some major work on the documentation package that caused things to move around. I completely forgot to check links in the Wiki. Sorry. I've fixed them now.

The new link to the reference pages for the javascript wrappers is https://github.khronos.org/KTX-Software/ktxjswrappers/index.html. The sub-link for the libktx wrapper is https://github.khronos.org/KTX-Software/ktxjswrappers/libktx_js.html. The main documentation page, https://github.khronos.org/KTX-Software will always be valid. Other things can be found from there.
